### PR TITLE
Add network free set

### PR DESCRIPTION
**Pull request by imamnurby**
This PR adds support for filtering instances that utilize networking functionality.

### CODE REVIEW

File: bigcodebench/data/network-free-set.txt
Comment (terryyz) [inline bigcodebench/data/network-free-set.txt:1]: Can we make this a constant.py file and import it to the other files?

Comment (terryyz) [reply bigcodebench/data/network-free-set.txt:1]: You can omit the `BigCodeBench/` part IMO

Comment (imamnurby) [reply bigcodebench/data/network-free-set.txt:1]: > Can we make this a constant.py file and import it to the other files?

Yes, it is possible.

> You can omit the `BigCodeBench/` part IMO

So, you mean we can have a constant.py and the content is the following:

```
OFFLINE_IDS = (
'1',
'2',
...

)
```

If the "BigCodeBench/" is removed, we should remove it too when getting the `task_id`.

```
json_data = [item for item in json_data if item.get("task_id") in included_ids]
...
hf_data = hf_data.filter(lambda instance: instance["task_id"] in included_ids)
```


What do you think?

Comment (imamnurby) [reply bigcodebench/data/network-free-set.txt:1]: @terryyz

Comment (terryyz) [reply bigcodebench/data/network-free-set.txt:1]: Yeah, you can strip off the `BigCodeBench/` in the `task_id` when iterating over the dataset :) I don't see an obvious issue here.

**Comment by imamnurby** [comment]
@terryyz i updated the PR

**Comment by terryyz** [comment]
merged to the network branch :)

**Pull request merged**